modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm

```diff
@@ -248,3 +248,22 @@
 	new /obj/item/grenade/spawnergrenade/manhacks(src)
 	for(var/i in 1 to 6)
 		new /obj/item/pen/edagger(src)
+
+/obj/item/storage/box/syndie_kit/loadout/greytide/PopulateContents() //hope you like reading baby, this kit's all about little initial power for massive endgame potential- you have staying power with energy weapons and other armaments but need to find or make them yourself, so GET TIDING
+	new /obj/item/spear/grey_tide(src)
+	new /obj/item/book/granter/crafting_recipe/pipegun_prime(src)
+	new /obj/item/book/granter/crafting_recipe/trash_cannon(src)
+	new /obj/item/construction/rcd/loaded/upgraded(src)
+	for(var/i in 1 to 3)
+		new /obj/item/rcd_ammo/large(src)
+	new /obj/item/clothing/under/color/grey/ancient(src)
+	new /obj/item/clothing/mask/gas(src)
+	for(var/i in 1 to 3)
+		new /obj/item/gas_filter(src)
```

Comment:
This is not really necessary to this kit and will just fill the box unnecessarily.

Reply:
true